standalone/ansible/roles/storage/tasks/main.yml

```diff
@@ -1,13 +1,25 @@
 # Sets up SSH keys on the storage server so that other servers can rsync.
+- name: generate ssh keys for remote user
+  openssh_keypair:
+    path: ~/.ssh/id_rsa
+
 - name: generate ssh keys for root user
   openssh_keypair:
     path: ~/.ssh/id_rsa
+  become: true
```

Comment:
Does `become`ing ensure that the root keys are placed somewhere else? Is that how the remote user and root user's keys are generated and fetched without confusion?

Reply:
We are generating keys for both root and the remote user here and then copying them both over. logrotate is run by the root user and it should be able to rsync as well.

The `become: true` is basically a `sudo su root` which means generate keys for `root`.